incubator/group/keeper.go

```diff
@@ -75,8 +75,8 @@ type Keeper struct {
 
 	// Vote Table
 	voteTable               orm.NaturalKeyTable
-	voteByProposalBaseIndex orm.UInt64Index
-	voteByVoterIndex        orm.Index
+	VoteByProposalBaseIndex orm.UInt64Index
+	VoteByVoterIndex        orm.Index
```

Comment:
What's the reason for making all of these public?

Reply:
I needed them available for tests. Indexes are read only and this should not create any harm.